src/main/java/com/sparta/spangeats/domain/store/service/StoreService.java

```diff
@@ -1,4 +1,50 @@
 package com.sparta.spangeats.domain.store.service;
 
+import com.sparta.spangeats.domain.member.entity.Member;
+import com.sparta.spangeats.domain.member.enums.MemberRole;
+import com.sparta.spangeats.domain.store.dto.StoreRequestDto;
+import com.sparta.spangeats.domain.store.dto.StoreResponseDto;
+import com.sparta.spangeats.domain.store.entity.Store;
+import com.sparta.spangeats.domain.store.exception.StoreLimitExceededException;
+import com.sparta.spangeats.domain.store.exception.AccessDeniedException;
+import com.sparta.spangeats.domain.store.exception.StoreNameAlreadyExistsException;
+import com.sparta.spangeats.domain.store.repository.StoreRepository;
+import jakarta.transaction.Transactional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+
+@Service
 public class StoreService {
+
+    private final StoreRepository storeRepository;
+    @Autowired
+    public StoreService(StoreRepository storeRepository) {
+        this.storeRepository = storeRepository;
+    }
+
+    // 1. 가게 생성 메소드
+    @Transactional
+    public StoreResponseDto createStore(StoreRequestDto storeRequestDto, Member member) {
+        // 사장님 권한 확인
+        if (!member.getRole().equals(MemberRole.OWNER)) {
+            throw new AccessDeniedException("사장님 권한이 필요합니다.");
+        }
```

Comment:
이런걸 entity한테 위임하는 게 좋을 거 같아요!! 아마 member의 role을 확인하는 로직은 다른데에도 쓰이기도 하고 service에서 꼭 확인할 필요 없어보이네요!! 
```suggestion
if (!isValidMemberRole){
    throw new AccessDeniedException("사장님 권한이 필요합니다.");
}
```
이렇게 변경하고 memeber 클래스에 isValidMemberRole라는 메서드를 만들어 그 안에 검증하는 방식인거죠!!